gen/_template/cfg.tmpl

```diff
@@ -243,6 +250,31 @@ func WithClient(client ht.Client) ClientOption {
 		}
 	})
 }
+
+{{ if $.EditorsEnabled -}}
+// RequestEditorFn  is the function signature for the RequestEditor callback function
+type RequestEditorFn func(ctx context.Context, req *http.Request) error
```

Comment:
Can you please drop the `Fn` suffix?
We don't plan to add interface-based editors, so we can just use `RequestEditor`.